src/unistd.rs

```diff
@@ -737,11 +737,23 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+/// Directive that tells lseek how to reposition the offset of the file.
 pub enum Whence {
+    /// The file offset shall be set to offset bytes.
     SeekSet,
+    /// The file offset shall be set to its current location plus offset.
     SeekCur,
+    /// The file offset shall be set to the size of the file plus offset.
     SeekEnd,
+    /// Adjust the file offset to the next location in the file greater than
+    /// or equal to offset containing data.  If offset points to data, then 
+    /// the file offset is set to offset.
     SeekData,
+    /// Adjust the file offset to the next hole in the file greater than or
```

Comment:
Again, this only "specifies", it doesn't "adjust". The `*seek()` functions adjust, these constants are just specifying a position.